CRC16CCITT.java

```diff
@@ -0,0 +1,36 @@
+// Computer Network Laboratory - CS58L
+// 5th Sem 2020-21
+// https://github.com/jssstu-lab/computer-networks
+
+// Write a program for error detecting code using CRC-CCITT (16- bits).
+
+import java.util.Scanner;
+
+public class CRC16CCITT {
+    public static void main(String[] args) {
+        int crc = 0xFFFF;
+        int polynomial = 0x1021;
+
+        Scanner sc = new Scanner(System.in);
+
+        String s = sc.next();
+
+        sc.close();
+
```

Comment:
Let us wait and check if she wants menu-driven or not
We can merge later tomorrow

Reply:
I agree with @aksharsramesh . I'll be pushing CPP programs as well.